pkg/sdn/plugin/egress_network_policy.go

```diff
@@ -108,8 +108,11 @@ func (plugin *OsdnNode) syncEgressDNSPolicyRules() {
 			continue
 		}
 
-		plugin.egressPoliciesLock.Lock()
-		defer plugin.egressPoliciesLock.Unlock()
-		plugin.updateEgressNetworkPolicyRules(vnid)
+		func() {
```

Comment:
Crash/panic will just exit and all locks vanish. Defer may be suitable if more lines of code are ever added in the future - less risk of any code running  without locks.